src/components/taskDetails/index.tsx

```diff
@@ -217,26 +219,21 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                                     />
                                 </div>
                             </TaskContainer>
-                            <TaskContainer
-                                title="Task DependsOn"
-                                hasImg={false}
-                            >
-                                <ol
-                                    className={
-                                        classNames[
-                                            'task_dependency_list_container'
-                                        ]
-                                    }
+                            {dev && (
```

Comment:
Instead of using dev, something more descriptive can be better here. Like `isTaskDependencyEnabled`

Reply:
done